scripts/ci-entrypoint.sh

```diff
@@ -62,10 +62,22 @@ setup() {
     if [[ -z "${CLUSTER_TEMPLATE:-}" ]]; then
         select_cluster_template
     fi
+    if [[ "${KUBERNETES_VERSION:-}" =~ "latest" ]]; then
+        CI_VERSION_URL="https://dl.k8s.io/ci/${KUBERNETES_VERSION}.txt"
+    fi
+    if [[ -n "${CI_VERSION_URL:-}" ]]; then
+        export CI_VERSION="${CI_VERSION:-$(curl -sSL "${CI_VERSION_URL}")}"
```

Comment:
for now this could just be in the block above after line 66 since `latest` is the only accepted pattern that would cause `CI_VERSION_URL` to be set

Reply:
Good idea.